apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -84,12 +84,31 @@ const styles = {
 const TutorialDetail = React.createClass({
   propTypes: {
     showing: React.PropTypes.bool.isRequired,
-    item: shapes.tutorial.isRequired,
+    item: shapes.tutorial,
     closeClicked: React.PropTypes.func.isRequired,
+    changeTutorial: React.PropTypes.func.isRequired,
     localeEnglish: React.PropTypes.bool.isRequired,
     disabledTutorial: React.PropTypes.bool.isRequired
   },
 
+  componentDidMount() {
+    document.addEventListener('keydown', this.onEscape);
+  },
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.onEscape);
+  },
+
+  onEscape({keyCode}) {
```

Comment:
this should just be something like onKeyDown right? we're not calling it just on escape